packages/homepage/content/docs/7-environment.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Environment'
+authors: ['Saeris']
+description: 'CodeSandbox uses different execution environments based on what type of Template you use.'
+---
+
+## What is an Environment?
+
+When using CodeSandbox, your code is evaluated and run by one of our built-in execution environments. These execution environments come in one of two types: Client and Container.
+
+Client environments run entirely inside of your browser and will continue to bundle your code even when you lose your connection to our servers. Containers on the other hand are executed server-side inside of a Docker container instance.
+
+While Client Sandboxes have faster previews and offline availability, they lack in configurability. Container Sandboxes take longer to start up, but effectively behave much like a local development environment, allowing you to customize your build tools, set up servers to listen to incoming requests, etc.
+
+Below is a list of each of our environments.
+
+## Client Environments
+
+Client Sandboxes each have their own bundler attached to them which are configured to support a specific framework and emulate their official CLI tools. They are not one-to-one implementations and thus cannot be configured within CodeSandbox, however they are designed to mirror the default behavior of the framework. If your project requires advanced configuration, try using a Container Sandbox instead.
```

Comment:
Great explanation, I want to mention that some client environments can be configured within CodeSandbox. We support some Parcel configuration and we support eg. custom babel config in CRA and other templates. Right now we word it in a way that suggests this is not possible, I think we should reword that bit.

Reply:
Hmm, okay yeah I can just change it to "and only support some minor configuration such as babel". Any per-bundler configuration options should be explained in their specific sections.